.github/workflows/tests.yml

```diff
@@ -34,27 +27,96 @@ jobs:
       with:
         context: .
         push: false
-        tags: appwrite-dev
+        tags: ${{ env.IMAGE }}
         load: true
         cache-from: type=gha
         cache-to: type=gha,mode=max
+        outputs: type=docker,dest=/tmp/${{ env.IMAGE }}.tar
         build-args: |
           DEBUG=false
           TESTING=true
           VERSION=dev
 
-    - name: Start Appwrite
-      run:  |
-        docker compose up -d
-        sleep 30
+    # - name: Pull Docker Images
+    #   run: docker compose pull --ignore-pull-failures --ignore-buildable
+    
+    # - name: List Files and images
+    #   run: |
+    #     docker compose config --images | sort -u
+    #     docker save $(docker compose config --images | sort -u) -o /tmp/allinone.tar
```

Comment:
If this is relevant, lets add comment and explain a little what it is. `tmp/allinone` is confusing me.

If not relevant, lets delete